src/meta/ClusterCheckHandler.h

```diff
@@ -0,0 +1,57 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+
+#ifndef META_CLUSTERCHECKHANDLER_H
+#define META_CLUSTERCHECKHANDLER_H
+
+#include "base/Base.h"
+#include <folly/executors/IOThreadPoolExecutor.h>
+#include "base/Status.h"
+#include "base/StatusOr.h"
+#include "thread/GenericWorker.h"
+#include "kvstore/KVStore.h"
+#include "meta/processors/admin/AdminClient.h"
+#include "meta/processors/Common.h"
+#include "meta/MetaServiceUtils.h"
+#include "ActiveHostsMan.h"
+
+namespace nebula {
+namespace meta {
+
+class ClusterCheckHandler {
```

Comment:
Just skim the cluster checker,  i think it is not what i wanted now.  
Please remove it in this pr and we use another pr for it,   it has no harm because we have "drop checkpoint" command. 

If create check point failed,  just drop it.